wiki/development/branching-and-release.md

```diff
@@ -10,28 +10,41 @@ order: 5
 
 ## 1 Versioning
 
-For ACE3 we use an versioning strategy based on <a href="http://semver.org/">Semver</a>. This means our version numbering is structured `MAJOR.MINOR.PATCH.BUILD`. 
+For ACE3 we use a versioning strategy based on <a href="http://semver.org/">Semver</a>. This means our version numbering is structured `MAJOR.MINOR.PATCH.BUILD`.
 
-Because this modification is for Arma and backwards compatability is not always possible, our `MAJOR.MINOR.PATCH.BUILD` rules are slightly different. We increment the:
+Because this modification is for Arma and backwards compatibility is not always possible, our `MAJOR.MINOR.PATCH.BUILD` rules are slightly different. We increment the:
 
-    MAJOR version when we switch to a new arma version (ie Arma 4 or standalone expansion),
-    MINOR version when we add new features or large amount of bug fixes.
-    PATCH version when a release only contains bug fixes.
+    MAJOR version when we switch to a new arma version (i.e. Arma 4 or standalone expansion).
+    MINOR version when a release contains significant changes or new features.
+    PATCH version when a release only contains fixes and enhancements.
 
-## 2 Branching and releases
+## 2 Project goals
 
-We have a release scheduled every 2 weeks on a Tuesday. On the Friday before release, the project management will decide whether or not this scheduled release will continue. When continuing  with the release, the current `master branch` will be merged into the `release branch`. The release branch will not contain any direct commits and no other branches will be merged into this branch. The exception being `hotfixes`, which are branched off `Release` and merged back into `Release` and `Master`. 
+Following the release of ACE 3.4.1, we have adopted a system involving collaborative project goals. Following `MINOR` releases the ACE development team will decide on a goal to work towards for the next minor increment (whether that be developing a new feature, overhauling an existing feature or simply enhancing a certain area of the project).
+
+As ACE is primarily a hobby project, project goals will never operate on a fixed deadline and nobody is required to contribute to them. They are simply a mechanism of directing the project's development between versions.
+
+Individual developers are free to continue working on the project at their own discretion and releases will otherwise operate as usual. Meaning that if features unrelated to the goal are complete, there will still be a minor version increment when the next release is made.
+
+Whenever a project goal is chosen:
+
+* A new issue will be made for the purpose of discussing and documenting the current goal's development in a centralized location.
+* A new branch originating from master will be made to develop collaboratively off of until the work is ready to be merged back in.
```

Comment:
This kind of goes against the mentality of creating changes that work independently. I feel like it would work a lot better to just continue with the previous way of doing things, but implement a way of directing progress towards the project goals.